csunplugged/static/scss/website.scss

```diff
@@ -101,16 +100,21 @@ div.plugged-in-language-implementation {
 $rounded-corner-radius: 0.5rem;
 
 .jumbotron {
-  background: linear-gradient( rgba($red,.7), rgba($red,.7) ), url("../img/home-page-banner-photo.jpg");
+  background: url("../img/red-hero-banner.png");
   background-size: cover;
-  border: $card-border-width solid darken($red, 10%);
-  border-radius: $rounded-corner-radius;
-  margin: 1rem 0;
   width: 100%;
+  padding-bottom: 15vh;
```

Comment:
Is there a reason for this much padding?

Reply:
I have it set ready for the cards to overlap the jumbotron.
![image](https://user-images.githubusercontent.com/20089275/33868344-42a21ddc-df67-11e7-9c70-b4d6050c4b92.png)

Reply:
Excellent.